src/resolvers.ts

```diff
@@ -1,19 +1,19 @@
 import * as AWS from 'aws-sdk';
-import { CloudFormation } from 'aws-sdk';
```

Comment:
I went ahead and removed this extra import. Assuming we want to utilize the wildcard across the board.